modules/nf-core/fastk/fastk/main.nf

```diff
@@ -33,6 +33,39 @@ process FASTK_FASTK {
         -N${prefix}_fk \\
         $reads
 
+    find . -name '*.ktab*' \\
```

Comment:
I find this very strange having to add this here. To me, this is an issue with the runner not running as the user.

Reply:
But the docker runner should be using the nf-test.config

https://github.com/nf-core/modules/blob/5e0c5677ea33b3d4c3793244035a191bd03e6736/tests/config/nf-test.config#L33

Reply:
I think you just found the problem. I believe it should be using:
https://github.com/nf-core/modules/blob/5e0c5677ea33b3d4c3793244035a191bd03e6736/tests/config/nf-test.config#L38 for the self hosted runners.

Reply:
The question is why are there two different settings for the same thing

Reply:
But if we remove user emulation, the module will not work in nf-core templated pipelines which use `-u $(id -u):$(id -g)`?

Reply:
Ah, sorry, I missed that. Yes, why? Good find.

Reply:
No, my point is that this should also use `-u $(id -u):$(id -g)` in the run options of the self hosted profile. And `fixownership` is something nextflow runs afterwards so shouldn't be used really. It's for user outputs

Reply:
`fixownership` is for when the container must run as `root`